cmd/version.go

```diff
@@ -5,20 +5,32 @@ package cmd
 
 import (
 	"fmt"
+	"runtime/debug"
 
 	"github.com/spf13/cobra"
 )
 
 // GoBlueprintVersion is the version of the cli to be overwritten by goreleaser in the CI run with the version of the release in github
 var GoBlueprintVersion string
 
+func getGoBlueprintVersion() string {
+	if len(GoBlueprintVersion) != 0 {
+		return GoBlueprintVersion
```

Comment:
We can probably define this inline and not have a separate variable declared on line 14

Reply:
But GoBlueprintVersion needs to be on package level to be able to be set by the ldflag in .goreleaser.yml?
Or am I misunderstanding you?